0008-S-sidecar-endpoint-tls.md

```diff
@@ -88,6 +101,9 @@ https://github.com/dapr/java-sdk/blob/76aec01e9aa4af7a72b910d77685ddd3f0bf86f3/s
 * Compatability guarantees
 This feature should allow localhost definition too `http://127.0.0.1:3500`, for example.
 
+this feature should continue to allow using other resolvers other than DNS (e.g.
```

Comment:
@artursouza  No released SDK currently supports this. If this is merged now, we don’t have to do any backwards compatibility support.

Reply:
We just released the python sdk that supports this last night.

Reply:
@JoshVanL this would never have been merged in time AND allowed enough time for PRs to be made, reviewers to review PRs / maintainers to merge them in time for the SDK release.

We have to be reasonable in the way we treat approvers and maintainers.

Reply:
Done :+1: